### PR TITLE
Feature/hide average buy price in position detail chart if no holding

### PR DESCRIPTION
**Pull request by dtslvr**

### CODE REVIEW

File: libs/ui/src/lib/line-chart/line-chart.component.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ export class LineChartComponent implements AfterViewInit, OnChanges, OnDestroy {
     const marketPrices = [];
 
     this.historicalDataItems?.forEach((historicalDataItem, index) => {
-      benchmarkPrices.push(this.benchmarkDataItems?.[index]?.value);
+      benchmarkPrices.push(this.benchmarkDataItems?.[index]?.value || null);
```
Comment (bptrgx) [inline libs/ui/src/lib/line-chart/line-chart.component.ts:126]: I wouldn't set null values in the line-chart component, as this component is not only used for the average unit price, but also for the net performance on the home page where zero shouldn't be hidden.

File: libs/ui/src/lib/line-chart/line-chart.component.ts [hunk 2]
```diff
@@ -154,7 +154,8 @@ export class LineChartComponent implements AfterViewInit, OnChanges, OnDestroy {
           data: benchmarkPrices,
           fill: false,
           label: this.benchmarkLabel,
-          pointRadius: 0
+          pointRadius: 0,
+          spanGaps: false
```
Comment (bptrgx) [inline libs/ui/src/lib/line-chart/line-chart.component.ts:158]: Shorter and easier to understand than my complex dataset split. 👍

File: libs/ui/src/lib/line-chart/line-chart.component.ts (hunk 1 above)
Comment (dtslvr) [reply libs/ui/src/lib/line-chart/line-chart.component.ts]: Good point

**Review by bptrgx — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**